removeDuplicates/removeDuplicates.js

```diff
@@ -5,4 +5,9 @@
  */
 const removeDuplicates = (arr) => {
 	//code here...
-};
\ No newline at end of file
+  return Array.from(new Set(arr));
```

Comment:
Clever clever. But I'd recommend finding a solution that shows you can solve this iteratively as well!